server/src/main/java/org/apache/iotdb/db/qp/physical/sys/LoadConfigurationPlan.java

```diff
@@ -115,6 +115,11 @@ public void deserialize(ByteBuffer buffer) {
     return null;
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    return null;
```

Comment:
`Collections.emptyList()`

Reply:
I'll change it. Thanks for your suggestion.